lib/ansible/plugins/become/su.py

```diff
@@ -71,6 +71,8 @@
             description:
                 - List of localized strings to match for prompt detection
                 - If empty we'll use the built in one
+                - Do NOT add a colon (:) to your custom entries. Ansible adds a colon at the end of each prompt;```
```

Comment:
Woopsie. I got those triple backticks into the text by mistake.
```suggestion
                - Do NOT add a colon (:) to your custom entries. Ansible adds a colon at the end of each prompt;
```